### ISSUE TITLE
Batch size and learning rate

### ISSUE THREAD

**Issue by parang99** [YueLiao/gen-vlkt#17]
Hi, thank you for sharing the great work. 

I wanted to reproduce the zero-shot setting and I got a question about batch size and learning rate. 
The default setting of [hico_s_zs_rf_uc.sh](https://github.com/YueLiao/gen-vlkt/blob/master/configs/hico_s_zs_rf_uc.sh) is a batch size of 2 and a learning rate of 1e-4 on 8 gpus.
Should I change the batch size or learning rate if I want to perform almost the same as the paper with 4 gpus instead of 8 gpus?
If so, I would appreciate it if you could specify the exact number. 

Thanks, 
Janet

**Comment by YueLiao** [comment]
Hi, thanks for your interest in our work!

Actually, we conduct most experiments based on 4 GPUs with a batch size of 4 at a 1e-4 learning rate. Thus, 4 GPUs are ok.

**Comment by parang99** [comment]
Thank for the quick reply.
There is another question about the zero-shot training. 

According to the config shell files, detr-r50-pre-2stage-hico.pth is used as pretrained model weight in [hico_s_zs_nf_uc.sh](https://github.com/YueLiao/gen-vlkt/blob/master/configs/hico_s_zs_nf_uc.sh), [hico_s_zs_rf_uc.sh](https://github.com/YueLiao/gen-vlkt/blob/master/configs/hico_s_zs_rf_uc.sh). And detr-r50-pre-2branch-hico.pth in [hico_s_zs_uo.sh](https://github.com/YueLiao/gen-vlkt/blob/master/configs/hico_s_zs_uo.sh), [hico_s_zs_uv.sh](https://github.com/YueLiao/gen-vlkt/blob/master/configs/hico_s_zs_uv.sh). 

In [README.md](https://github.com/YueLiao/gen-vlkt#pre-trained-model), there is no how to make detr-r50-pre-2stage-hico.pth file. 

Are detr-r50-pre-2stage-hico.pth and detr-r50-pre-2branch-hico.pth exactly same but just different only the name?

**Comment by YueLiao** [comment]
> Are detr-r50-pre-2stage-hico.pth and detr-r50-pre-2branch-hico.pth exactly same but just different only the name?

”Are detr-r50-pre-2stage-hico.pth and detr-r50-pre-2branch-hico.pth exactly same but just different only the name?“ Yes
